src/test/testdata/expected/lsl/AVLTreeArkTsFunSignatures.lsl

```diff
@@ -0,0 +1,56 @@
+libsl "1.1.0";
+library std version "11" language "Java" url "https://gitee.com/openharmony/arkcompiler_runtime_core/blob/master/static_core/plugins/ets/stdlib/std/containers/AVLTree.ets";
```

Comment:
I strongly suggest removing any mention of third parties, especially trademarks and other IPs (intellectual properties).

```suggestion
library std version "11" language "unknown" url "-";
```

Reply:
Fixed